fvm/env.go

```diff
@@ -550,22 +549,6 @@ func (e *transactionEnv) GetComputationLimit() uint64 {
 }
 
 func (e *transactionEnv) CreateAccount(payer runtime.Address) (address runtime.Address, err error) {
-	if e.ctx.ServiceAccountEnabled {
-		err = e.vm.invokeMetaTransaction(
-			e.ctx,
-			deductAccountCreationFeeTransaction(
-				flow.Address(payer),
-				e.ctx.Chain.ServiceAddress(),
-				e.ctx.RestrictedAccountCreationEnabled,
-			),
-			e.ledger,
-		)
-		if err != nil {
-			// TODO: improve error passing https://github.com/onflow/cadence/issues/202
-			return address, err
-		}
-	}
```

Comment:
I joined this together with adding flow token vault/receiver and adding the storage reservation receiver. Which means that we do one meta transaction for everything, instead of two separate ones.